zstash/globus.py

```diff
@@ -158,6 +158,10 @@ def file_exists(name: str) -> bool:
     return False
 
 
+# TODO: What does gv stand for? Globus something? Global variable?
```

Comment:
^

Reply:
Global Variable.  If I must use them, I like to label them as such.

Reply:
Good idea.  That might discourage people from using them - should be a standard!